### PR TITLE
README.md and Community-version.py

### PR DESCRIPTION
**Pull request by MehulGupta21**
Sir, I'm one of your students. please accept my pull request

### CODE REVIEW

File: community-version.py [hunk 1]
```diff
@@ -3,6 +3,7 @@
 #code credit goes to: https://www.hackerearth.com/practice/notes/beautiful-python-a-simple-ascii-art-generator-from-images/
 #code modified to work with Python 3 by @aneagoie
 from PIL import Image
+example/make_art.py/ztm-logo.png
```
Comment (monde16) [inline community-version.py:6]: This line is gonna break the program. Why is it necessary?

Comment (ndeoligence) [inline community-version.py:6]: @MehulGupta21, this line is gonna break the program. I think you meant to add a comment?

Comment (MehulGupta21) [reply community-version.py:6]: Srry yeah you are right

**Review by sophiabrandt — CHANGES_REQUESTED** [review]
Please apply the changes that @ndeoligence suggested.

**Comment by MehulGupta21** [comment]
Changed, please merge

**Review by sophiabrandt — CHANGES_REQUESTED** [review]
Please run the function `python3 community-version.py example/ztm-logo.png` and fix any errors that pop up.
I'm still getting an error:
```python
 File "community-version.py", line 7
    =======
    ^
SyntaxError: invalid syntax
```

**Comment by MattCSmith** [comment]
Closing due to the age of the unresolved conflict. 
Feel free to resolve and reopen.

**Pull request closed without merging**